lizard_damage/forms.py

```diff
@@ -318,6 +328,26 @@ def clean(self):
         """Checks that apply to more than one field."""
         cleaned_data = super(FormStep1, self).clean()
 
+        if not cleaned_data['customheights']:
+            # get needed AHN files
+            ahn_files = calculation._get_ahn_leaves(
+                cleaned_data['waterlevel_dataset'], logger)
+            ahn_files = [ahn[0] for ahn in ahn_files]
+            logger.debug('Required AHN %s files: %s',
+                         cleaned_data['ahn_version'],
+                         ahn_files)
+            # Check that AHN-files exists for selected AHN-version
+            ahn_data_dir = os.path.join(settings.LIZARD_DAMAGE_DATA_ROOT,
+                                        'data_ahn' + cleaned_data['ahn_version'])
+            for file in ahn_files:
+                file_name = file + '.tif'
+                prefix = file_name[1:4]
+                path = os.path.join(ahn_data_dir, prefix, file_name)
+                if not os.path.isfile(path):
+                    logger.debug("file %s not present", file_name)
+                    self.add_field_error('ahn_version', 'Geen AHN-kaart beschikbaar '
```

Comment:
Die impliciete string-concatenaties maken me altijd wat zenuwachtig. Zoiets is duidelijker:

    self.add_field_error(
        'ahn_version', 
        'Geen AHN-kaart beschikbaar voor het gebied van deze waterstand.')